code/modules/materials/Mat_Processing.dm

```diff
@@ -87,6 +87,10 @@
 			S.satchel_updateicon()
 			return
 
+		if (W.cant_drop) //For borg held items
+			user.show_text("You can't put that in [src] when it's attached to you!", "red")
```

Comment:
Prefer `boutput` to `show_text`, I think? (See example a few lines down)

I'm unsure of the differences at this time, but I see `boutput` a _lot_ more.

Reply:
this is cargo cult'd from a similar fix I did to kitchen stuff a few weeks ago. if you feel strongly though I can change it